samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/ItemFacadeQueries.java

```diff
@@ -94,6 +94,7 @@ public void deleteItem(Long itemId, String agent) {
             }
             getHibernateTemplate().merge(item);
         }
+        getHibernateTemplate().delete(item);
```

Comment:
Should this be inside the `if` block? What happens when you call `.delete()` on a `null` item?